apis/apps/v1alpha1/cluster_types.go

```diff
@@ -776,7 +776,8 @@ type ClusterComponentSpec struct {
 	// with other Kubernetes resources, such as modifying Pod labels or sending events.
 	//
 	// Defaults:
-	// If not specified, KubeBlocks automatically assigns a default ServiceAccount named "kb-{cluster.name}",
+	// If not specified, and if any of the role probe, volume protection, and data protection features are enabled,
+	// KubeBlocks automatically assigns a default ServiceAccount named "kb-{cluster.name}",
 	// bound to a default role installed together with KubeBlocks.
```

Comment:
change to this:

// To perform certain operational tasks, agent sidecars running in Pods require specific RBAC permissions.
// The service account will be bound to a default role named "kubeblocks-cluster-pod-role" which is installed together with KubeBlocks.
// If not specified, KubeBlocks automatically assigns a default ServiceAccount named "kb-{cluster.name}"